azurerm/resource_arm_postgresql_server.go

```diff
@@ -138,7 +138,6 @@ func resourceArmPostgreSQLServer() *schema.Resource {
 						"storage_mb": {
 							Type:         schema.TypeInt,
 							Required:     true,
-							ForceNew:     true,
```

Comment:
Can we update the docs to mention that this is no longer ForceNew? (And above)

Reply:
Yeah, I thought about that after I did the initial push. I'll update those now.

Reply:
I just looked at the documentation and the original text does not mention that it will ForceNew so I believe this is a no op:

```Markdown
* `storage_mb` - (Required) Max storage allowed for a server. Possible values are between `5120` MB(5GB) and `1048576` MB(1TB) for the Basic SKU and between `5120` MB(5GB) and `4194304` MB(4TB) for General Purpose/Memory Optimized SKUs. For more information see the [product documentation](https://docs.microsoft.com/en-us/rest/api/mysql/servers/create#StorageProfile).
```